xmake/rules/c++/modules/modules_support/clang/compiler_support.lua

```diff
@@ -209,7 +209,11 @@ function get_stdmodules(target)
                 -- @see https://github.com/llvm/llvm-project/pull/76451 (has been revert, so we need to wait)
                 local clang_path = path.directory(get_clang_path(target))
                 local clang_lib_path = path.join(clang_path, "..", "lib")
-                local modules_json_path = find_file("*libc++.modules.json", clang_lib_path)
+                local modules_json_path = find_file("libc++.modules.json", clang_lib_path)
```

Comment:
why not use path.join + os.isfile directly here?

Reply:
because i'm dumb :D